caravel/models.py

```diff
@@ -755,6 +758,7 @@ def all_view_names(self, schema=None):
             pass
         return views
 
+    @utils.memoized
```

Comment:
Ah, my fault, confused it with flask memoized, that one lives within the request. I'll delete it for meanwhile.